### DISCUSSION TITLE
Erreur sur les mises à jour

### DISCUSSION THREAD

**Discussion by Lb59** [dbarzin/deming#68] [General]
Bonjour,
Lorque je lance la mise à jour composer sans etre root j'ai deux erreurs sur la cmd composer self-update >> "Command "Self-update  is not defined" et  pour le composer update une erreur de permission sur le fichier ligne 872 de Filesystem.php "failed to open stream sur /vendor/composer/installed.php). Je n'ai en effet plus ce message sour root.
Par contre composer deconseille update avec le compte root .
Que dois je faire ? merci

**Comment by dbarzin** [comment]
Quel OS et quelle version de l'OS utilises-tu ?

Il doit y avoir un problème de droits :

    sudo chown -R $USER /var/www/deming/*

**Comment by dbarzin** [comment]
See https://github.com/dbarzin/deming/issues/69
